waterdata/templates/monitoring_location.html

```diff
@@ -6,6 +6,16 @@
 {% block extra_head_tags %}
     {% if status_code == 200 %}
         <meta name="description" content="{{ components.Description(stations[0].site_no, location_with_values, parm_grp_summary) }}">
+        <!-- tags for Facebook Open Graph -->
+        <meta property="og:url" content="{{ url_for('monitoring_location', site_no=site_no, _external=True) }}" />
```

Comment:
Can you use  stations[0].siteno here rather than explicitly passing in the site_no in the context?